app/views/groups/index.html.erb

```diff
@@ -49,6 +49,11 @@
     }) %>
 <% end %>
 
+<br /><br />
+<h1 class="faq_heading"><%= t('pages.faq.group_question') %></h1>
+<br />
```

Comment:
I added this extra `<br />` because for some weird reason I didn't get an extra line break after the `</h1>` tag